content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -2,85 +2,103 @@
 Start Receiving Tickets
 =======================
 
-Offering a variety of channels from where your customers can contact you grants them flexibility
-and the right to choose the best one for themselves. And, in order to make sure inquiries across
-all channels get addressed, it is essential to have a solution where all interactions come in one
-place.
+Helpdesk offers multiple channels where customers can reach out for assistance. In order to ensure
+all inquiries are addressed in a timely fashion, it is essential that all interactions are received
+in one central location.
 
-Channels options to submit tickets
+Channel options to submit tickets
 ==================================
 
-Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`, and enable the following
-features as you want them to be available to your users.
+Go to :menuselection:`Helpdesk --> Configuration --> Teams --> Channels` to select which
+channel options to enable. The available channels are *Email Alias*, *Website Form*, and
+*Live Chat*.
 
-.. image:: receiving_tickets/channels_options.png
+.. image:: receiving_tickets/receivingtickets_channeloptions.png
    :align: center
    :alt: View of a helpdesk teams setting page emphasizing the channels options in Odoo Helpdesk
 
 Email Alias
 -----------
 
-| Let your customers submit tickets by sending an email to your support email address. The subject
-  line of the email becomes the title of the ticket and the content is shown in the Chatter.
-| Select *Configure domain name* to be redirected to *Settings* and, from there, enable *External
-  Email Servers* to determine or change your *Alias Domain*.
+With this setting, users can create a ticket by sending an email to a custom email alias. The
+subject line of the email becomes the title of the ticket and the body of the message is added to
+the Chatter.
 
-.. image:: receiving_tickets/channels_emailalias.png
+Select :guilabel:`Configure a custom domain` to be redirected to :guilabel:`Settings` and, from
+there, enable :guilabel:`Custom Email Servers` to determine or change an :guilabel:`Alias Domain`.
+Click :guilabel:`Save`.
```

Comment:
This section is way too short.

Provide _all_ of the information readers are going to need to be able to successfully accomplish their goals. Be as specific as possible by _showing_ (not _telling_) them exactly what to do and where to go. Use time-order language to guide the reader from one step to the next until they're done with their task.

```suggestion
To set a helpdesk team's custom email alias, navigate to the :guilabel:`Teams` form and look below
the :menuselection:`Channels --> Email Alias` heading. With the :guilabel:`Email Alias` setting
enabled, then select the :guilabel:`Configure a custom domain` link to navigate to the
:menuselection:`General Settings`. Then, under the :guilabel:`Discuss` heading in the settings, find
and enable the :guilabel:`Custom Email Servers` option to determine or change an :guilabel:`Alias
Domain`.

.. example::
   A simple (and typical) value for an :guilabel:`Alias Doman` would be a company's name, or the
   subdomain's prefix value when the Odoo database was initially set up, e.g., `stealthywood`, so
   the base alias address would be ``stealthywood.odoo.com``. Then, after the :guilabel:`Alias
   Domain` is set, custom email aliases for different teams may be applied, such as
   ``support@stealthywood.odoo.com``.

.. note::
   The configuration steps outlined above are for **Odoo Online** and **Odoo.sh** users. For
   **On-premise** users, additional configuration for custom email servers and email aliases might
   be needed. Read more information on :ref:`Send and receive emails in Odoo with an email server
   <email_communication/default_from>`.

When the :guilabel:`Alias Domain` setup is complete, click :guilabel:`Save`, and then navigate
back to the helpdesk :guilabel:`Teams` form. Now, an :guilabel:`Alias` label will be visible under
the :guilabel:`Email Alias` setting. Next to the label, is an blank field where the helpdesk team's
custom email alias can be entered.
```

Reply:
I did update and expand on this section, however, I think the process you are describing is from Runbot, where you have to configure the custom domain. Since runbot is such a specific case, this process really isn't as universal (most databases would already have the domain already configured. I ran several tests with new demo dbs to confirm with different configurations).